examples/sklearnex/spmd_linear_regression.py

```diff
@@ -0,0 +1,71 @@
+#===============================================================================
+# Copyright 2023 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#===============================================================================
+
+import numpy as np
+
+from mpi4py import MPI
+
+import dpctl
+
+from numpy.testing import assert_allclose
+
+from onedal.spmd.linear_model import LinearRegression as LinRegSpmd
+
+from sklearn.datasets import make_regression
```

Comment:
Group imports without too many empty lines

Reply:
Flake 8 will not be satisfied

Reply:
Done.